test/controllers/admin/users_controller_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class Admin::UsersControllerTest < ActionController::TestCase
+  attr_reader :user
+
+  setup do
+    @user = users(:admin)
+
+    sign_in_as(@user)
+  end
+
+  test "PUT #approve a given user" do
+    lawyer = Lawyer.create!(name: 'New Lawyer', email: 'new@layer.com', password: 'password')
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.